static/app/views/issueList/container.tsx

```diff
@@ -1,33 +1,56 @@
-import {Component} from 'react';
+import React, {Component} from 'react';
 import DocumentTitle from 'react-document-title';
 
 import LightWeightNoProjectMessage from 'app/components/lightWeightNoProjectMessage';
 import GlobalSelectionHeader from 'app/components/organizations/globalSelectionHeader';
-import {Organization} from 'app/types';
+import GroupStore from 'app/stores/groupStore';
+import {Organization, Project} from 'app/types';
+import {callIfFunction} from 'app/utils/callIfFunction';
 import withOrganization from 'app/utils/withOrganization';
+import SampleEventAlert from 'app/views/organizationGroupDetails/sampleEventAlert';
 
 type Props = {
   organization: Organization;
+  projects: Project[];
 };
 
 class IssueListContainer extends Component<Props> {
```

Comment:
@Neo-Zhixing Can we pass `State` as a secondary arg to the generic? So like:

````
Component<Props, State>
````